### PR TITLE
Add matomo tracker for marketplace

### PR DESCRIPTION
**Pull request by goreck888**
This PR contains integration with PC matomo tracking service.
Service pages visits, orders and ratings are marked as matomo events:
- Category of events => "Service"
- Order/AddToProject => "AddToProject"
- Service rate => "Rate"
- Page visit => "Visit"
With event name => service ID recognizable by PC applications
For rating event value is `ServiceOpinion.service_rating`

For test case you need:
- visit a few services pages
- add to project service
- rate a service

Then I'll check if all cases make events for matomo

### CODE REVIEW

File: app/views/layouts/_head.html.haml [hunk 1]
```diff
@@ -9,5 +9,6 @@
   = favicon_link_tag asset_path("favicon-180x180.png"), rel: "apple-touch-icon", type: "image/png"
   = stylesheet_pack_tag "application"
   = javascript_pack_tag "application"
+  = render "layouts/matomo"
```
Comment (mkasztelnik) [inline app/views/layouts/_head.html.haml:12]: I propose one small improvement here. 

```haml
- if Rails.env.production?
  = render "layouts/matomo"
  = render "layouts/google_analytics"
￼  = render "layouts/hotjar_trackingcode"
```

File: app/views/layouts/_matomo.html.haml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+%script
+:javascript
+  var _paq = _paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="//www.einfracentral.eu/matomo/";
```
Comment (mkasztelnik) [inline app/views/layouts/_matomo.html.haml:8]: Maybe this can be extracted to ENV variable?

File: app/views/services/_header.html.haml [hunk 3]
```diff
@@ -30,6 +30,7 @@
     - if policy(service).order?
       = link_to t(".order"),
                     service_offers_path(service),
-                    class: "btn btn-primary d-block mb-3"
+                    class: "btn btn-primary d-block mb-3",
+                    onclick: "_paq.push(['trackEvent', 'Service', 'Order']);"
```
Comment (mkasztelnik) [inline app/views/services/_header.html.haml:34]: This does not mean that service was ordered, use only starts ordering process - is it ok?

File: app/views/layouts/_head.html.haml (hunk 1 above)
Comment (goreck888) [reply app/views/layouts/_head.html.haml:12]: I want to add it when everything will works fine.

File: app/views/services/_header.html.haml (hunk 3 above)
Comment (goreck888) [reply app/views/services/_header.html.haml:34]: After consultation with @agpul we need to count real orders, so it's bad placed trigger.

File: app/views/layouts/_matomo.html.haml (hunk 2 above)
Comment (goreck888) [reply app/views/layouts/_matomo.html.haml]: Done.

**Comment by agpul** [comment]
I've tested. Please check and report if anything changes.

File: app/helpers/order_helper.rb [hunk 4]
```diff
@@ -14,4 +14,16 @@ def order_nav_link(title, path, state)
       end
     end
   end
+
+  def order_submit_options(step_key, orderable, id)
```
Comment (martaswiatkowska) [inline app/helpers/order_helper.rb:18]: Where do you use this?

Comment (goreck888) [reply app/helpers/order_helper.rb:18]: I forget about deleting it. Thanks.

**Pull request merged**